src/components/modules/blockManager.ts

```diff
@@ -237,16 +237,16 @@ export default class BlockManager extends Module {
     data = {},
     id = undefined,
     tunes: tunesData = {},
-  }: {tool: string; id?: string; data?: BlockToolData; tunes?: {[name: string]: BlockTuneData}}): Block {
+  }: {tool: string; data?: BlockToolData; tunes?: {[name: string]: BlockTuneData};id?: string}): Block {
     const readOnly = this.Editor.ReadOnly.isEnabled;
     const tool = this.Editor.Tools.blockTools.get(name);
     const block = new Block({
-      id,
       data,
       tool,
       api: this.Editor.API,
       readOnly,
       tunesData,
+      id,
```

Comment:
Order matters nothing here, because it the object, so it would better to leave it first I guess.